src/and-cli-restore-azure-storage.ts

```diff
@@ -0,0 +1,165 @@
+                generate one on its own cadence.
+            */
+            const options = {
+                deleteDestination: deleteDestination,
+                destination: {
+                    account: destinationAccount,
+                    container: destinationContainer,
+                    sasToken: destinationSasToken,
+                },
+                recursive: recursive,
+                source: {
+                    account: sourceAccount,
+                    container: sourceContainer,
+                    sasToken: sourceSasToken,
+                },
+            };
+
+            AzureAzcopySync.containers(options);
+
+            Echo.newLine();
+            Echo.success(
+                "Containers successfully synced in Azure Blob Storage!"
+            );
+        },
+        validateOrExit() {
+            const errors = [];
+
+            if (program.deleteDestination != null) {
+                deleteDestination = program.deleteDestination;
+            }
+            if (program.recursive != null) {
+                recursive = program.recursive;
+            }
+
+            // Validate arguments
+            destinationAccount = program.destinationAccount;
+            if (destinationAccount == null) {
+                errors.push("--destination-account is required");
+            }
+
+            destinationContainer = program.destinationContainer;
+            if (destinationContainer == null) {
+                errors.push("--destination-container is required");
+            }
+
+            destinationSasToken = program.destinationSasToken;
+            if (destinationContainer == "" || destinationContainer == null) {
+                errors.push("--destination-sas-token is required");
+            }
+
+            sourceAccount = program.sourceAccount;
+            if (sourceAccount == null) {
+                errors.push("--source-account is required");
+            }
+
+            sourceContainer = program.sourceContainer;
+            if (sourceContainer == null) {
+                errors.push("--source-container is required");
+            }
+
+            sourceSasToken = program.sourceSasToken;
+            if (sourceContainer == "" || sourceContainer == null) {
+                errors.push("--source-sas-token is required");
+            }
+
+            // Bail if up-front arguments are errored
+            if (errors.length > 0) {
+                Echo.errors(errors);
+                shell.exit(1);
+            }
+
+            return true;
+        },
+    };
+
+    // #endregion Functions
+
+    // -----------------------------------------------------------------------------------------
+    // #region Entrypoint
+    // -----------------------------------------------------------------------------------------
+
+    program
+        .usage("option")
+        .description(restoreAzureStorage.description())
+        .option(
```

Comment:
Nitpick: Consider alphabetizing the list of options by the flag names

Reply:
Wait, they're not alphabetized?

Reply:
Please excuse me, my morning coffee must not have kicked in yet 😅 . Yep, they're definitely alphabetized.